README.md

```diff
@@ -14,52 +13,56 @@ Omnibus is a unified data bus which manages the connection of various data sourc
 
 ### Installation
 
 1. Clone this repo
-   - If you have git configured with SSH, run `git clone git@github.com:waterloo-rocketry/omnibus.git`
-   - If you don't have git configured with SSH (or you're not sure what that means), run `git clone https://github.com/waterloo-rocketry/omnibus.git`
 2. Enter the newly-cloned repo with `cd omnibus`
-3. Create a virtual environment:
-   - First install the virtualenv library: `pip install virtualenv`
-   - Create the venv: `python -m venv venv`
-4. Activate the virtual environment:
-   - For Mac/Linux: `source venv/bin/activate`
-   - For Windows using Git Bash: `source venv/Scripts/activate`
-5. There are two ways to install Omnibus. The setup script is the easiest method, but if it doesn't work you can manually install the required packages
-   1. Setup script:
-      - For Mac/Linux or Windows Git Bash: `source setup.sh`
-   2. Manual installation:
-      - Upgrade pip version: `pip install --upgrade pip`
-      - Run `pip install wheel`, which will help install the rest of the packages more quickly
-      - Install Python dependencies with `pip install -r requirements.txt`. You'll also need to run that command in each of the following folders:
-        - `sources/ni/`
-        - `sources/parsley/`
-        - `sinks/dashboard/`
-        - If you get a permission error, try `pip install --user -r requirements.txt` instead.
-      - Install the Omnibus library locally with `pip install -e .`
-        - Don't forget the `.`! This allows the sources and sinks (and you) to import Omnibus
-      - Initialize the `Parsley` submodule with `git submodule update --init --recursive` and install the library locally with `pip install -e ./parsley`
+3. In a terminal, run `uv python install`
+4. Run `uv sync --locked --all-packages`
+
+### Update
+
+1. Run `git pull`
+2. Run `uv sync --locked --all-packages`
 
 ## Usage
 
-Omnibus works by running sources (to send data), sinks (to receive data) and a server to connect them together.
-The easiest way to run Omnibus is through the launcher script `python launcher.py`. This will open a GUI (or prompt by text if you pass in the `--text` flag) where you can select the sources and sinks to run. You can also manually run the different components of Omnibus using the commands described below.
+To use Omnibus typically, you'll need at minimum a server and the dashboard running. You will probably also want to connect it to one or many data sources. Run each in a separate terminal window.
 
-Known limitation of the launcher script: currently, CLI flags can't be passed to a source/sink so one requiring them (such as `parsley`) need to be run manually.
+Run the following commands at the base of the omnibus folder.
 
 ### Server
 
-Start the Omnibus server by activating your `venv`, then running `python -m omnibus`.
+`uv run omnibus-server`
+
+### Dashboard
+
+`uv run src/dashboard/main.py`
```

Comment:
I don't think this necessarily installs the dependencies there, like `pyqtgraph`

Reply:
skill issue on my part :sob: please ignore this, you're right